modules/shadertools/src/modules/picking/picking.js

```diff
@@ -20,7 +20,11 @@ function getUniforms(opts = DEFAULT_MODULE_OPTIONS) {
     }
   }
   if (opts.pickingHighlightColor) {
-    uniforms.picking_uHighlightColor = opts.pickingHighlightColor.map(x => x / 255);
+    const color = opts.pickingHighlightColor.map(x => x / 255);
+    if (!Number.isFinite(color[3])) {
+      color[3] = 1;
```

Comment:
This won't work if `color` is a typed array (do we care about that)? An alternative approach:
```js
if (color.length < 4) {
    color = new Float32Array(4).set(color);
    color[3] = 1;
}
```

Reply:
Good catch!